Sources/MapboxCoreNavigation/RouteController.swift

```diff
@@ -464,11 +471,15 @@ open class RouteController: NSObject {
         delegate?.router(self, didUpdate: progress, with: location, rawLocation: rawLocation)
         
         //Fire the notification (for now)
-        NotificationCenter.default.post(name: .routeControllerProgressDidChange, object: self, userInfo: [
-            NotificationUserInfoKey.routeProgressKey: progress,
-            NotificationUserInfoKey.locationKey: location, //guaranteed value
-            NotificationUserInfoKey.rawLocationKey: rawLocation, //raw
-        ])
+        
+        var userInfo: [RouteController.NotificationUserInfoKey: Any] = [
+            .routeProgressKey: progress,
+            .locationKey: location, // guaranteed value
+            .rawLocationKey: rawLocation, // raw
+        ]
+        userInfo[.headingKey] = heading
```

Comment:
Above, there’s a `rawHeading` constant too, but it isn’t set here. Does this code need to distinguish between a raw heading set by Core Location and one that’s somehow cleaned up by MapboxNavigationNative? My understanding is that MapboxNavigationNative only cleans up courses, not headings. However, if we were to introduce that functionality at some point, changing the behavior of `headingKey` could be considered backwards-incompatible.

/ref https://github.com/mapbox/mapbox-navigation-ios/pull/2215#discussion_r328379605